src/nodes/NodeConnection.ts

```diff
@@ -464,9 +466,11 @@ class NodeConnection<M extends ClientManifest> {
   }) {
     this.validatedNodeId = validatedNodeId;
     this.nodeId = nodeId;
-    this.host = host;
+    this.host = quicUtils.toCanonicalIP(host) as unknown as Host;
```

Comment:
Is this because the 2 `Host` doesn't overlap?

Reply:
If we eventually abstract a network types and utilities like `js-network`, we could eventually unify all the types and utilities since alot of different libraries depend on it.

Reply:
Yeah, I have to convert to the quic host and back.